### PR TITLE
add new slang content:

### PR DESCRIPTION
**Pull request by shajyhia**
SearchAndReplace
randomNumberGenerator

### CODE REVIEW

File: org/openscore/slang/base/utilities/randomNumberGenerator.sl [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+# Outputs:
+#   - returnResult - A random number between max and min (inclusive)
+##################################################
+
+namespace: org.openscore.slang.base.utilities
+operations:
+  - randomNumberGenerator:
+      inputs:
+        - min
```
Comment (meirwah) [inline org/openscore/slang/base/utilities/randomNumberGenerator.sl]: maybe add defualt value of 0 for min input?

File: org/openscore/slang/base/strings/searchAndReplace.sl [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      action:
+        python_script: |    
+          if replace in input:
+            returnResult = input.replace(replace, replaceWith)
+          else:
+            returnResult = input 
+      outputs:
+        - returnResult
+      results:
+        - SUCCESS
```
Comment (meirwah) [inline org/openscore/slang/base/strings/searchAndReplace.sl]: maybe add rule/regex for SUCCESS/FAILURE ? like for illegal argument?

**Comment by meirwah** [comment]
looks good! :+1:

File: org/openscore/slang/base/utilities/randomNumberGenerator.sl (hunk 1 above)
Comment (shajyhia) [inline org/openscore/slang/base/utilities/randomNumberGenerator.sl]: modified to allow negative range

Comment (shajyhia) [inline org/openscore/slang/base/utilities/randomNumberGenerator.sl]: Now negative numbers is allowed

**Pull request merged**